adminrestrict/middleware.py

```diff
@@ -218,13 +221,17 @@ def process_request(self, request):
             if self.request_ip_is_allowed(request):
                 return None
 
-            if self.disallow_get:
+            if self.disallow_get and self.denied_template:
+                return render(request, self.denied_template, {})
+            elif self.disallow_get:
                 return HttpResponseForbidden(self.denied_msg)
             else:
                 return None
 
         if restricted_request_uri and request.method == 'POST':
-            if not self.request_ip_is_allowed(request):
+            if not self.request_ip_is_allowed(request) and self.denied_template:
+                return HttpResponseForbidden(self.denied_template)
```

Comment:
Does this one need to be render() too?

Reply:
Yes, this returns the html webpage if the denied_template is configured and the access is denied.

Reply:
I don't understand the difference between line 225 and this one. In line 225 we render() a page with template, but here we pass template to HttpResponseForbidden().